TestPlatform.sln

```diff
@@ -929,6 +931,18 @@ Global
 		{186069FE-E1E8-4DE1-BEA4-0FF1484D22D1}.Release|x64.Build.0 = Release|Any CPU
 		{186069FE-E1E8-4DE1-BEA4-0FF1484D22D1}.Release|x86.ActiveCfg = Release|Any CPU
 		{186069FE-E1E8-4DE1-BEA4-0FF1484D22D1}.Release|x86.Build.0 = Release|Any CPU
+		{62E9D32B-B989-43CF-81A2-B38B3367FCA3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{62E9D32B-B989-43CF-81A2-B38B3367FCA3}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{62E9D32B-B989-43CF-81A2-B38B3367FCA3}.Debug|x64.ActiveCfg = Debug|Any CPU
```

Comment:
@MarcoRossignoli Maybe we could also clean the TP sln to remove the extra/un-needed entries.

Reply:
we can follow-up on this...we did it the same on CC repo...